### PR TITLE
Release/v1.0.0 beta.12

### PR DESCRIPTION
**Pull request by QuintinWillison**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+
+**Implemented enhancements:**
+
+- Add log statements to both SDKs [\#524](https://github.com/ably/ably-asset-tracking-android/issues/524)
+- Display resolution information in the example apps [\#510](https://github.com/ably/ably-asset-tracking-android/issues/510)
+- Send resolution to subscriber [\#509](https://github.com/ably/ably-asset-tracking-android/issues/509)
+
+**Fixed bugs:**
+
+- Calling remove\(\) should remove a trackable that isn't finished being added [\#450](https://github.com/ably/ably-asset-tracking-android/issues/450)
```
Comment (KacperKluka) [inline CHANGELOG.md:15]: It looks like the parentheses were escaped with backslashes and we've ended up with `remove\(\)`, we should remove it :wink: 
```suggestion
- Calling remove should remove a trackable that isn't finished being added [\#450](https://github.com/ably/ably-asset-tracking-android/issues/450)
```

**Review by KacperKluka — CHANGES_REQUESTED** [review]
All good, just found a small typo :wink:

**Review by owenpearson — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (KacperKluka) [reply CHANGELOG.md:15]: @QuintinWillison feel free to resolve this conversation when you make the fix (as I might not be around and I don't want to block the release) :wink:

Comment (QuintinWillison) [reply CHANGELOG.md:15]: As it happens, it doesn't actually [render](https://github.com/ably/ably-asset-tracking-android/blob/release/v1.0.0-beta.12/CHANGELOG.md) like that. But, equally, you're right that these don't need escaping so I shall remove that.

Comment (QuintinWillison) [reply CHANGELOG.md]: https://github.com/ably/ably-asset-tracking-android/pull/531/commits/3c9e32d8a6c3c313d29fd2009f0895aa0a30a737

**Review by ikbalkaya — APPROVED** [review]

**Review by AndyNicks — APPROVED** [review]

**Pull request merged**